evm-ds/src/main.rs

```diff
@@ -98,44 +93,123 @@ impl CallContext {
     }
 }
 
+#[derive(Debug,Serialize,Deserialize)]
+struct StructLog {
+    pub depth: usize,
+    pub error: String,
+    pub gas: u64, // not populated
+    #[serde(rename = "gasCost")]
+    pub gas_cost: u64, // not populated
+    pub op: String,
+    pub pc: usize,
+    pub stack: Vec<String>,
+    pub storage: Vec<String>, // not populated
+}
+
+impl StructLog {
```

Comment:
Maybe you could use Default derivation on the struct?, and then use `StructLog::default()` instead of `new()`

Reply:
Yup, sounds good!